web/styles/subscriptions.css

```diff
@@ -171,6 +171,7 @@ h4.user_group_setting_subsection_title {
             }
 
             .subscriber_list_remove {
+                padding-right: 18px;
```

Comment:
I guess `16px` should be enough here, since the scrollbar width on hovering over it is `15px`.